### PR TITLE
OCTO-10653 Django 3

### PR DESCRIPTION
**Pull request by lorandvarga**
Updates gitignore.
Updates pre-commit dependencies.
Removes unused travis config.
Updates runtests.py
Removes setup.py and tox and trasitions to pyproject.toml.
Adds license file.
Moving towards using docker-compose for tests.
Renouncing python3.6 support since it's dead. RIP. Updates test case due to Django 3.1.2 change.

### CODE REVIEW

File: .github/workflows/prod.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+      - name: Publish test results
+        uses: actions/upload-artifact@master
+        with:
+          name: test-results
+          path:
+            test-results_*
+        if: ${{ always() }}
+
+      - name: Upload package to pipy
```
Comment (radu-dan) [inline .github/workflows/prod.yaml]: typo: should be pypi instead of pipy

File: .github/workflows/test.yaml [hunk 2]
```diff
@@ -31,7 +31,7 @@ jobs:
 
       - name: Upload package to testpipy
```
Comment (radu-dan) [inline .github/workflows/test.yaml]: typo: should be pypi instead of pipy

File: README.rst [hunk 3]
```diff
@@ -11,10 +11,12 @@ This app requires:
     - for Django < 1.9      please use django-atris < 1.0.0
     - for Django < 1.10     please use django-atris < 1.2.0
     - for Django > 2.0.0    please use django-atris > 1.2.1
+    - for Django > 3.2      please use django-atris >= 2.0.1
 - Postgresql
 - Python:
     - for django-atris < 2.0.0  please use Python >= 2.7 or Python >= 3.4 (after Django 2)
     - for django-atris >= 2.0.0 please use Python >= 3.6
```
Comment (radu-dan) [inline README.rst:18]: i think = instead of >= might be cleaner

File: .github/workflows/prod.yaml (hunk 1 above)
Comment (lorandvarga) [reply .github/workflows/prod.yaml]: thanks

File: .github/workflows/test.yaml (hunk 2 above)
Comment (lorandvarga) [reply .github/workflows/test.yaml]: right

File: README.rst (hunk 3 above)
Comment (lorandvarga) [reply README.rst:18]: yeah, good point

**Review by radu-dan — APPROVED** [review]

**Pull request merged**